python/GafferUI/SpreadsheetUI/_ProxyModels.py

```diff
@@ -164,6 +164,24 @@ def __dataChanged( self, topLeft, bottomRight, roles ) :
 
 		self.dataChanged.emit( proxyTopLeft, proxyBottomRight, roles )
 
+	def __headerDataChanged( self, orientation, first, last ) :
+
+		if orientation == QtCore.Qt.Vertical :
+			limit = self.rowCount() - 1
+			offset = self.__startRow
+		else :
+			limit = self.columnCount() - 1
+			offset = self.__startColumn
+
+		# Don't propagate if the changed range is ourside of our remapping
```

Comment:
ourside -> outside

Reply:
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaah :exploding_head: What is is with these letters! Sorry, fixed [here](https://github.com/GafferHQ/gaffer/commit/5bee5f1e3eec2bb8460911a7fda8cd0fc9a21e14#diff-b36bcf1dcccb9d58d9085cd12f153f0ff3cc57f6f02cd769219bbd70c77608c6R176).